pxr/usd/usdLux/lightAPI.h

```diff
@@ -384,7 +555,20 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // COLOR 
     // --------------------------------------------------------------------- //
-    /// The color of emitted light, in energy-linear terms.
+    /// The color of emitted light, in the rendering color space.
+    /// 
+    /// This color is just multiplied with the emission:
+    /// 
+    /// <center><b>
+    /// L<sub>Color</sub> = L<sub>Scalar</sub> ⋅ color
+    /// </b></center>
+    /// 
+    /// In the case of a spectral renderer, this color should be uplifted such
+    /// that it round-trips to within the limit of numerical accuracy under the
+    /// rendering illuminant.  We recommend the use of a rendering color space
+    /// well defined in terms of a Illuminant D illuminant, to avoid unspecified
```

Comment:
do we wanna go further and just say D50 and D65 to be in line with common practice? The canonicals include D55 and D75 as well, but I'm not aware that they're used anywhere that might also use UsdLux. The canonical illumininants are magical because the use the newer value of Planck's constant. If we just say a "D" illuminant, it makes me think we might recommend 1931 D illuminants, in addition to the four "magic" illuminants.

Reply:
Yeah that seems reasonable